package.json

```diff
@@ -78,7 +79,7 @@
     "escodegen": "^2.0.0",
     "eslint": "^8.57.0",
     "eslint-config-prettier": "^9.1.0",
-    "eslint-plugin-simple-import-sort": "^12.0.0",
+    "eslint-plugin-import": "^2.29.1",
```

Comment:
What was the rationale for changing to this over the previous plugin?

Reply:
The previous plugin doesn't handle import duplicates, the new ones does